crypto/async/arch/async_posix.h

```diff
@@ -48,7 +49,11 @@
  */
 #   define USE_SWAPCONTEXT
 #  endif
-#  include <ucontext.h>
+#  if defined (OPENSSL_SYS_TANDEM)
+#   include <tdmsig.h>
+#  else
+#   include <ucontext.h>
+#  endif
```

Comment:
This won't get compiled because this part of the code is inside `!defined(OPENSSL_SYS_TANDEM)` condition. So please either drop this change or the change above.